### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by TatumCreative** [nfroidure/gulp-iconfont#74]
It would be helpful if there is a human readable changelog (I'm not seeing it if there is one). I'm migrating an earlier setup and am having issues figuring out what's different with the new versions.

Thanks for your work!

**Comment by nfroidure** [comment]
You are absolutely right ;). Leaving this issue open until i find some time to add automatic changelog to my projects.

**Comment by igidas** [comment]
+1

**Comment by pkyeck** [comment]
was wondering about the same thing. I have two projects (one using v8 the other v9) and I think the CSS output is different for the same SVGs using the same template ... is that expected?

**Comment by nfroidure** [comment]
As said in #169 : A changelog is still a good thing but I won't have time to add it that soon. Feel free to PR that said. Would merge if it's done automagically on commit. For an example, see my recenter modules : https://github.com/nfroidure/streamqueue

**Comment by MiniGod** [comment]
Similar story here. I was upgrading from an old version and had to look through the commits to see where stuff would break.

Here are the breaking changes that mattered to me:

`v10.0.0`: upgrades to gulp 4 & requires minimum node v6.
`v11.0.0`: Requires minimum node v12.

If you are, like me, upgrading an old project that is still using gulp@3, upgrade to version 9.

**Comment by MiniGod** [comment]
Also worth noting that in `v3.0.0` the `codepoints(codepoints, options)` event was changed to `glyphs(glyphs, options)`, as seen in the [diff for v3.0.0](https://github.com/nfroidure/gulp-iconfont/compare/v2.0.0...v3.0.0). And `codepoints[].codepoint` changed to `glyphs[].unicode[0]`. I can't say I understand the change at first glance. These are 6 year old changes, so hopefully you don't have to bother with that.
